phovea_security_flask/flask_login_impl.py

```diff
@@ -39,6 +39,7 @@ def logout(self, user):
 
 class NamespaceLoginManager(security.SecurityManager):
   def __init__(self):
+    print('init flask login')
```

Comment:
Please use `_log.info(...)`

Reply:
I will remove that - was only for an internal test